Multisig/UI/ClaimToken/Select Guardian/GuardianListViewController.swift

```diff
@@ -120,17 +126,28 @@ class GuardianListViewController: LoadableViewController {
 extension GuardianListViewController: UISearchResultsUpdating, UISearchBarDelegate, UISearchControllerDelegate {
 
     func updateSearchResults(for searchController: UISearchController) {
-        guard let resultsController = searchController.searchResultsController as? GuardianSearchResultController else {
-            return
-        }
-        if !searchController.searchBar.text!.isEmpty {
+        if !isSearchBarEmpty {
             Tracker.trackEvent(.userClaimChdelSearch)
+
+        } else if !isSearchBarEmpty && filteredGuardians.isEmpty {
+            Tracker.trackEvent(.screenClaimChdelNf)
         }
+
         let terms = searchController.searchBar.text!
             .trimmingCharacters(in: .whitespacesAndNewlines)
             .components(separatedBy: " ") as [String]
 
-        resultsController.filteredGuardians = guardians.filter { guardian in
+        filterContentForSearchText(terms)
+        makeSections(items: isFiltering ? filteredGuardians : guardians)
```

Comment:
why is this distinction whether result set is being filtered needed?

Reply:
filtered guardians could always be used and in case of unfiltered state filtered guardians = unfiltered guardians. This way some if checks could be saved.